src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/ReferralController.kt

```diff
@@ -227,7 +227,7 @@ constructor(
           ""
         }
       },
-      hasConfirmation = chosenStatus.hasConfirmation,
+      hasConfirmation = chosenStatus.hold != true && chosenStatus.hasConfirmation == true,
```

Comment:
Minor suggestion: Since its all boolean fields

`!chosenStatus.hold && chosenStatus.hasConfirmation`

Reply:
as they are both nullable Boolean they need the ==true bit